src/pages/index.js

```diff
@@ -9,12 +9,14 @@ export default function Home() {
   const { livestreamUrl } = useSiteMetadata()
   
   return (
-    <Layout>
-      <Player
-        streamUrl={livestreamUrl}
-        preloadType="auto"
-      />
-      <Calendar />
+    <Layout className={isPlayerLoaded && isCalendarLoaded ? "ready" : ""}>
```

Comment:
I would rather encapsulate the className logic inside Layout.js, and prefer passing a LayoutProp such as areChildrenLoaded, or something? WDYT?